modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -1043,6 +1063,10 @@ Type::
 | `string`
 | logLevel of the collector runtime
 
+| `logTypes`
+| `string`
+| logTypes defines the desired record types to generate. Possible values are "FLOWS" (default) to export flowLogs, "CONVERSATIONS" to generate newConnection, heartbeat, endConnection events, "ENDED_CONVERSATIONS" to generate only endConnection events or "ALL" to generate both flow logs and conversations events
```

Comment:
It's perhaps a bit tricky, because this CRD doc is generated from the go sources and has 3 outputs:
- this asciidoc file
- the upstream markdown
- the openapi description that is embedded into the CRD - and shows up, for instance, in the OLM form.
So we tend to play conservative in the formatting, to make sure we don't break any of these three.

Reply:
but we can try and check nothing is broken

Reply:
That does seem tricky to implement just in this table cell :/ I can see how the list would be cleaner. I think these options are also covered well in list-form elsewhere in the doc, so we could leave it as is here I think. I can see how it would be kinda wonky to generate a list here.

Reply:
The only other option is, I just edit directly this table cell (which technically it is supposed to be a copy/paste, but I've had to directly edit a few things like OpenShift -> {product-title}

Reply:
no worries, if it's too tricky then should be okay I think to leave it as you have.